### PR TITLE
Android: Add Pause / Unpause Emulation Toggle

### PR DESCRIPTION
**Pull request by Ebola16**
Implements https://bugs.dolphin-emu.org/issues/10602 and should survive orientation changes.

- [x] Confirm `fragment_ingame_menu.xml` changes for Android TV work.

Screenshot_20200608-120650_Dolphin Emulator

### CODE REVIEW

**Comment by Ebola16** [comment]
Reminder to update `NativeLibrary.UnPauseEmulation` conditions if #8833 is merged first (or vice versa).

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java [hunk 1]
```diff
@@ -80,12 +85,14 @@
   private String mSelectedGameId;
   private int mPlatform;
   private String[] mPaths;
+  private static boolean sUserPausedEmulation;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:88]: This should not be static.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java [hunk 2]
```diff
@@ -70,6 +70,11 @@
 
   private Settings mSettings;
 
+  private MenuItem mPauseEmulationButton;
+  private MenuItem mUnpauseEmulationButton;
+  private View mPauseEmulationTv;
+  private View mUnpauseEmulationTv;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:76]: To me, it would make more sense if these four didn't exist and we just called `findItem`/`findViewById` in the functions that need to access these. It's a bit hard to follow where and how these get initialized the way it is now.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java (hunk 1 above)
Comment (Ebola16) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:88]: `EmulationActivity.hasUserPausedEmulation()` uses `sUserPausedEmulation` and is called in EmulationFragment so don't we need a static variable for `EmulationActivity.hasUserPausedEmulation()`?

Comment (JosJuice) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:88]: You should be able to use `activity.hasUserPausedEmulation()` in EmulationFragment.

Comment (Ebola16) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:88]: But in EmulationFragment, class `EmulationState` is static and `hasUserPausedEmulation()` is called within that class. `activity` isn't static (and can't be static) so wouldn't my code be the best approach here?

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java (hunk 2 above)
Comment (Ebola16) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:76]: Just to be clear, calling `findItem/findViewById` in both `showPauseEmulationButton()` and `showUnpauseEmulationButton()` while having both of those methods take `Menu` as an argument is more readable? That would duplicate the `findItem/findViewById` for each method and make me add a variable to pass `Menu`, which seems worse to me.

I did find that I can group the variable's assignments together though.

Comment (JosJuice) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:76]: Oh, I had missed the part that it was called on `menu` and not just the activity... I suppose it makes sense the way it currently is, then.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java (hunk 1 above)
Comment (JosJuice) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java:88]: Ah, I had missed that. But can't we just make EmulationState non-static, or pass in `activity` when creating an EmulationState?

It is preferable to not use static state in a way that would cause things to break when creating two instances of a class at the same time, even if we never actually do that with EmulationActivity.

Comment (Ebola16) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java]: Let's be safe and pass `activity`

**Comment by Ebola16** [comment]
Something went wrong while testing. Looking into it now.

**Comment by Ebola16** [comment]
@JosJuice passing `activity` to `EmulationState` didn't seem to work after orientation changes and `EmulationState` has a static inner class so I think it would be best to go with the static variable here. Tested.

**Comment by Ebola16** [comment]
@JosJuice Thanks! That helped and I think I got it working on Android TV but I still can't truly test it since I can't initialize OGL on the Android Emulator. Another check would be appreciated. Marking this as working on Android TV.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java [hunk 3]
```diff
@@ -21,11 +22,17 @@
 
 public final class MenuFragment extends Fragment implements View.OnClickListener
 {
+  private View mPauseEmulationTv;
+  private View mUnpauseEmulationTv;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java:26]: These two don't need to have `Tv` in the name anymore. (Feel free to ignore this if you want to.)
